### PR TITLE
Use real locale addons file for locale recommender

### PR DESCRIPTION
**Pull request by maurodoglio**

### CODE REVIEW

File: taar/recommenders/locale_recommender.py [hunk 1]
```diff
@@ -17,13 +16,15 @@ class LocaleRecommender:
     may not work.
     """
     def __init__(self):
-        self.top_addons_per_locale = fetch_json(ADDON_LIST_PER_LOCALE_URL)
+        self.top_addons_per_locale = utils.get_s3_json_content(ADDON_LIST_BUCKET,
+                                                               ADDON_LIST_KEY)
+        print(self.top_addons_per_locale)
```
Comment (Dexterp37) [inline taar/recommenders/locale_recommender.py]: I think that's a leftover :-D

File: taar/recommenders/locale_recommender.py [hunk 2]
```diff
@@ -17,13 +16,15 @@ class LocaleRecommender:
...
 
     def can_recommend(self, client_data):
         # We can't recommend if we don't have our data files.
         if self.top_addons_per_locale is None:
             return False
-
+        print("is not none")
```
Comment (Dexterp37) [inline taar/recommenders/locale_recommender.py]: This one too!

File: taar/recommenders/locale_recommender.py [hunk 3]
```diff
@@ -1,9 +1,8 @@
-from .utils import fetch_json
+from ..recommenders import utils
 
-# TODO: this should be a public s3 bucket IFF we are allowed
-# to share this info publically.
-ADDON_LIST_PER_LOCALE_URL = \
-    'http://www.thiswillonedaybearealurl.com/amazing_things.json'
+
+ADDON_LIST_BUCKET = 'telemetry-private-analysis-2'
```
Comment (Dexterp37) [inline taar/recommenders/locale_recommender.py]: Smart idea! I like this one ;)

File: taar/recommenders/locale_recommender.py (hunk 1 above)
Comment (maurodoglio) [inline taar/recommenders/locale_recommender.py]: Argh!

File: taar/recommenders/locale_recommender.py (hunk 2 above)
Comment (maurodoglio) [inline taar/recommenders/locale_recommender.py]: Aaaargh!

**Pull request merged**